app/src/main/java/com/example/music_app/domain/repositories/PlaylistsRepository.kt

```diff
@@ -5,6 +5,8 @@ import com.example.music_app.data.models.ListOfPlaylists
 import com.github.michaelbull.result.Result
 import kotlinx.coroutines.flow.Flow
 
+private const val TYPE = "playlist"
```

Comment:
`Minor` I would rename it to smth more meaningful. Also, since this parameter could be passed from the outside maybe it would be better to make this constant public and move somewhere else, what do you think about that ?